src/SFA.DAS.AANHub.Application.UnitTests/Profiles/Queries/ProfileModelTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.AANHub.Application.Profiles.Queries.GetProfiles;
+using SFA.DAS.AANHub.Domain.Entities;
+
+namespace SFA.DAS.AANHub.Application.UnitTests.Profiles.Queries
+{
+    [TestFixture]
+    public class ProfileModelTests
+    {
+        [Test]
+        public void ProfileModelTest_ReturnsExpectedFields()
+        {
+            const string category = "Events";
```

Comment:
Could use AutoData attribute to generate random values for Profile

Reply:
Implemented